src/main/java/at/hannibal2/skyhanni/utils/RenderUtils.kt

```diff
@@ -410,6 +437,25 @@ object RenderUtils {
         GuiEditManager.add(this, posLabel, longestX, offsetY)
     }
 
+    enum class AlignmentEnum {
+        LEFT, CENTER, RIGHT
+    }
```

Comment:
can you change this to (and) move it to the top, Line 37):
reference: #885 
```suggestion
    enum class HorizontalAlignment{LEFT, CENTER, RIGHT}
```